### DISCUSSION TITLE
Path Item Object $ref - how does it work?

### DISCUSSION THREAD

**Discussion by mabar** [OAI/OpenAPI-Specification#3045] [Q&A] [answered]
I understand that Reference Object is for reusability of parts of the schema. Everywhere where reference is allowed is written that type can be either Reference Object or the other object type.

Except in Path Item Object. In Path Item Object are allowed the standard fields and $ref which references another Path Item Object? It also has type string instead of Reference Object and I found description really confusing.

Is it just inconsistency or does $ref in Path Item Object work differently? I would expect that Callback Object and Paths Object would both accept list of Path Item Object | Reference Object.

Thank you

**Comment by mabar** [comment, marked as answer]
For now I just merge data from original and referenced Path Item and throw an exception in case field exists in both, because behavior is undefined by specification

**Comment by handrews** [comment]
Looks like this was answered.  There are also clarifications and changes relevant to this coming for 3.1.1, 3.2.0, etc.:

* #2655 
* #2656 
* #2657
